src/test/regress/citus_tests/config.py

```diff
@@ -293,14 +294,22 @@ def __init__(self, arguments):
         self.new_settings = {
             "citus.task_assignment_policy": "first-replica",
             "citus.enable_fast_path_router_planner": False,
-            "citus.enable_local_execution": False,
             "citus.enable_single_hash_repartition_joins": True,
             "citus.recover_2pc_interval": "1s",
             "citus.remote_task_check_interval": "1ms",
             "citus.values_materialization_threshold": "0",
         }
 
 
+class CitusUnusualQuerySettingsNoLocalExecConfig(CitusUnusualQuerySettingsConfig):
```

Comment:
hmm, why do we inherit from `CitusUnusualQuerySettingsConfig`? Shouldn't we inherit from `CitusDefaultClusterConfig`?

Also, I'm a little confused now, why do we need enable_local_execution = False when use_citus_managed_tables = false? I though the other way is needed. When use_citus_managed_tables = True, we cannot have enable_local_execution = false.


But, when use_citus_managed_tables = false, we don't care enable_local_execution? Am  I missing this?

Reply:
I just wanted to keep `CitusUnusualQuerySettingsConfig` config tested.
`CitusUnusualQuerySettingsNoLocalExecConfig` now behaves like the old `CitusUnusualQuerySettingsConfig`; while the new `CitusUnusualQuerySettingsConfig` is a new unusual config but with `use_citus_managed_tables` set to on

Reply:
Ah, I see.

I think in general `CitusUnusualQuerySettingsConfig` is not very interesting to have these two variations. Given "the time to execute" angle, I'd probably drop `CitusUnusualQuerySettingsNoLocalExecConfig` and move "citus.use_citus_managed_tables": False, to `CitusUnusualQuerySettingsConfig`.  

Unless you have a good/important reason to keep?

(such as https://github.com/citusdata/citus/compare/add-citus-managed-tables-to-arbitrary-configs-onder?expand=1)

Reply:
No important reason, just wanted to prevent our test coverage from decreasing, by testing these unusual configurations both with citus_managed_tables is on and off

Reply:
Well, as I said, this test is not interesting enough to run multiple times. I'm not concerned about the small coverage we lost. So, I'd rather go with my above suggestion.

I think this is too much: `by testing these unusual configurations both with citus_managed_tables is on and off`
What is enough is: `by testing these with citus_managed_tables is on and off`

My point is that we are planning to expand arbitrary config tests coverage significantly, and any non-very-useful schedule adds quite a bit of time.